mailscanner/languages/en.php

```diff
@@ -934,6 +934,7 @@
     'message61' => 'The variable %s is empty, please set a value in conf.php.',
     'text611' => 'Quarantine Report for %s',
     'text612' => 'In the last %s day(s) you have received %s e-mails that have been quarantined and are listed below. All messages in the quarantine are automatically deleted %s days after the date that they were received.',
+    'text613' => 'In the last %s day(s) you have received no e-mails that have been quarantined.',
```

Comment:
For fr.php

'text613' => 'Durant le(s) %s dernier(s) jour(s) vous avez reçu %s messages qui ont été mis en Quarantaine.',

Reply:
done